Session/Conversations/ConversationVC+Interaction.swift

```diff
@@ -1611,7 +1611,19 @@ extension ConversationVC:
                 quotedAttachment.downloadUrl == Attachment.nonMediaQuoteFileId,
                 let quotedInteraction = try? quote.originalInteraction.fetchOne(db)
             {
-                let attachment = try? quotedInteraction.attachments.fetchAll(db).first
+                let attachment: Attachment? = {
+                    if let attachment = try? quotedInteraction.attachments.fetchAll(db).first {
```

Comment:
You can probably just `fetchOne(db)` here instead of `fetchAll(db).first` (the result _should_ be the same but it'll avoid fetching and decoding the rest of the attachments which should be more efficient)